recipes/testbook/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "testbook" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3aef6aa44de9accf62839e8c9f471fd558add64eca64056729dea08824589756
```

Seems to be not a valid URL... this also does not list any tar.gz sdist file: https://pypi.org/project/testbook/#files

This should work instead:

```suggestion
  url: https://github.com/nteract/{{ name }}/archive/{{ version }}.tar.gz
  sha256: f35f66f3c074cb05e3d51000a5aaf7355a48826e29365f15b9681c0e998f35da
```